Update Angular page (#1025)
jhipster/jhipster.github.io: pages/using_angular.md

## Project Structure

> The JHipster client code can be found under `src/main/webapp`, and follows closely the  [Angular style guide](https://github.com/johnpapa/angular-styleguide/blob/master/a2/README.md). Please read this guide first if you have any question on our application structure, file names, TypeScript conventions...

Reviewer: There's probably an Angular official guide we should link to.

Reviewer 2: https://angular.io/guide/styleguide

Author: Oh yes indeed, I thought we already were linking the official one !

## Using Angular CLI

Angular CLI is used to build and test JHipster applications. 

> However, we added a custom Webpack configuration file in order to improve Developer Experience and User Experience.

Reviewer: ```suggestion
However, we added a custom webpack configuration file in order to improve the developer experience.
```

How? What does JHipster's webpack config do that Angular CLI doesn't?

Author: The main thing is that we add BrowserSync, ESLint (Angular CLI is still on TSLint for now), merging JSON translation files and also the notify plugin

Reviewer 2: What about this wording?

> However, we added a custom webpack configuration file in order to improve the developer experience by adding BrowserSync, ESLint (Angular CLI is still on TSLint for now), merging JSON translation files. So, rather than running `ng build` you should run `npm run build`, while `ng serve` should be replaced by `npm start` .

Author: Well ng build and ng serve work out of the box because I pointed our custom Webpack configuration in the `angular.json` so I think the last part isn't necessary, WDYT ?

Author: Here is the wording i purpose

> However, we added a custom webpack configuration file in order to improve the developer experience by adding BrowserSync, ESLint (Angular CLI is still on TSLint for now), merging JSON translation files and add notifications when build has completed or failed.
